File: backend/routes/feedback_routes.go
package routes

import (
	"eunity.com/backend-main/controllers"
	"github.com/gin-gonic/gin"
)

var Feedback_controllers = controllers.Feedback_controllers{}

func Feedback_routes(r *gin.RouterGroup) {
	r.POST("/add", Feedback_controllers.Add_feedback)
	r.GET("/get", Feedback_controllers.Get_feedback)
	r.GET("/getall", Feedback_controllers.Get_all_feedback)
}

File: python/routes/feedback_routes.py
from fastapi import APIRouter, Depends
from controllers import feedback_controllers as feedback_controller


router = APIRouter()

@router.post("/add")
async def add_feedback():
    return feedback_controller.add_feedback()

@router.get("/get")
async def get_feedback():
    return feedback_controller.get_feedback()

@router.get("/getall")
async def get_all_feedback():
    return feedback_controller.get_all_feedback()